lean4/src/putnam_1962_b6.lean

```diff
@@ -4,11 +4,13 @@ open BigOperators
 open MeasureTheory
 
 theorem putnam_1962_b6
-(π : ℝ := Real.pi)
+(π : ℝ)
+(hπ : π = Real.pi)
```

Comment:
`open scoped Real` makes this unecessary.

Reply:
Thanks! This was done in 2 problems which should be amended shortly.